interfaces/builtin/network_manager.go

```diff
@@ -67,6 +67,29 @@ network packet,
 
 /dev/rfkill rw,
 
+# Needed for modem connections using PPP
+/usr/sbin/pppd ix,
+/etc/ppp/** rwix,
+/dev/ppp rw,
+/dev/tty[^0-9]* rw,
+/run/udev/data/* r,
+/proc/sys/kernel/modprobe r,
+/bin/kmod ix,
+capability sys_module,
```

Comment:
I'd prefer to have snapd add the rules rather than using a Ux/ux rule on pppd. This just shifts the attack from modem manager to pppd.